object/domain.go

```diff
@@ -88,6 +88,10 @@ func UpdateDomain(ctx context.Context, id string, domain *Domain) (bool, error)
 		return false, err
 	}
 
+	if len(domain.GetId()) > policyMaxValueLength {
+		return false, fmt.Errorf("id too long for policies")
```

Comment:
This is invalid error message, i believe. This code updates domain

Reply:
why wrong? code updates domain and throne "id too long" so you understand that id of domain too long. (id = owner/name

Reply:
will it better if I replace "id" for "owner/name " ?